docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.8"
```

Comment:
Could you check if this is the latest version? If so, please provide a link to the reference.